src/azurelinux/3.0/net9.0/cross/riscv64/Dockerfile

```diff
@@ -3,8 +3,7 @@ ARG ROOTFS_DIR=/crossrootfs/riscv64
 
 # Install the latest debootstrap
 RUN tdnf remove -y debootstrap && \
-    git clone --depth 1 "https://git.launchpad.net/ubuntu/+source/debootstrap" /debootstrap && \
-    chmod a+x /debootstrap/debootstrap && \
+    curl -sSL https://deb.debian.org/debian/pool/main/d/debootstrap/debootstrap_1.0.134.tar.gz | tar xzf - -C / && \
```

Comment:
would you mind adding sha checksum validation like we do e.g. here:
https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/0ca583b11fc3fc618335bf9394d79daba6884739/src/azurelinux/3.0/net9.0/cross/armv6/Dockerfile#L4-L10